### DISCUSSION TITLE
5.2.6 - SMTP Connect() Failed

### DISCUSSION THREAD

**Discussion by drewjohnstone** [PHPMailer/PHPMailer#2991] [General]
Hi all,
I've recently taken over a custom site, and its using this mailer to send out password resets etc. We are trying to get it to connect SMTP to gmail however I keep getting 

SMTP -> ERROR: Failed to connect to server: Connection timed out (110)SMTP Connect() failed. Message could not be sent.Mailer Error: SMTP Connect() failed.

Can anyone suggest anything to get this sending mails again please?

Edit: *unnecessary code removed*

**Comment by Synchro** [comment]
The code you posted is just a very old copy (from 2013) of this library, and posting it here won't explain anything (other than that you *really* need to get a handle on your dependencies!). We would need to see your code that drives it and the debug output.
It doesn't help that you ignored the requests when you created this ticket to a) update to the latest version, and b) read the troubleshooting guide which covers [this exact issue](https://github.com/PHPMailer/PHPMailer/wiki/Troubleshooting#smtp-error-could-not-connect-to-smtp-host). Please read that, do the tests that it suggests, read [the hundreds of answers to this same question on Stack Overflow](https://stackoverflow.com/search?q=%5Bphpmailer%5D+SMTP+connect+failed), and if you still can't figure it out, ask on there.
